app/assets/stylesheets/elements/_inputs.scss

```diff
@@ -0,0 +1,5 @@
+input[type=number]::-webkit-inner-spin-button,
+input[type=number]::-webkit-outer-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
```

Comment:
Cribbed from [this article on CSS Tricks](https://css-tricks.com/snippets/css/turn-off-number-input-spinners/).